.changeset/silly-lemons-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/blocks-admin": patch
```

Comment:
Do you think this is a patch or a minor worthy change?

Reply:
I think patch is okay